auth/email_action_links.go

```diff
@@ -32,6 +32,7 @@ type ActionCodeSettings struct {
 	AndroidMinimumVersion string `json:"androidMinimumVersion,omitempty"`
 	AndroidInstallApp     bool   `json:"androidInstallApp,omitempty"`
 	DynamicLinkDomain     string `json:"dynamicLinkDomain,omitempty"`
+	ReturnOobLink         bool   `json:"returnOobLink"`
```

Comment:
API Review feedback was to rename this variable to `SendEmailLink` in order to maintain the default value false default value for boolean

Reply:
@Dal-Papa Would you be able to make this change, please?

Reply:
As you want but that will break the override that was happening on the key name due to the `toMap` function...